packages/flutter/lib/src/rendering/selection.dart

```diff
@@ -98,6 +98,16 @@ abstract class SelectionHandler implements ValueListenable<SelectionGeometry> {
   /// See also:
   ///  * [SelectionEventType], which contains all of the possible types.
   SelectionResult dispatchSelectionEvent(SelectionEvent event);
+
+  /// Gets the TextSelection from the selected content
+  ///
+  /// Return `null` if nothing is selected, or is not supported by the selectable
+  TextSelection? getLocalTextSelection();
```

Comment:
How would one use this information if there isn't an API to know the content?

Reply:
I think this is a valid point, and maybe it does need a function to get the content. At least for my use case (and I assume other's who'd need this type of information) already know's their content and their content's offsets that they're displaying. Perhaps a separate PR if this becomes a need?